### PR TITLE
fix(static files): issue when non-linked file was not served

### LINKED ISSUE #135: Static file is not served if `link=False`
After #131 lona don't add static files with `link=False` to HTML. But also it stopped serving them.

### PR DESCRIPTION
**Pull request by maratori**
Fixes #135

### CODE REVIEW

File: lona/static_file_loader.py [hunk 1]
```diff
@@ -170,15 +166,15 @@ def discover(self):
         self.style_tags_html = self.server.templating_engine.render_template(
             self.server.settings.STATIC_FILES_STYLE_TAGS_TEMPLATE,
             {
-                'stylesheets': [i for i in self.node_stylesheets if i.enabled],
+                'stylesheets': [i for i in self.node_stylesheets if i.enabled and i.link],  # NOQA: LN001
```
Comment (fscherf) [inline lona/static_file_loader.py:169]: Could you put this in a separate line or add a newline?

File: tests/test_static_files.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    @app.route('/')
+    class MyLonaView(LonaView):
+        def handle_request(self, request):
+            return 'SUCCESS'
+
+
+async def test_static_files(lona_app_context):
+    some_existing_file = os.path.basename(__file__)
+
+    # isn't discovered in setup_app()
```
Comment (fscherf) [inline tests/test_static_files.py:20]: What does this mean?

**Review by fscherf — CHANGES_REQUESTED** [review]
@maratori Great changes! In general i am ok with this, just some nitpicking

Thanks for adding tests! :)

File: tests/test_static_files.py (hunk 2 above)
Comment (maratori) [reply tests/test_static_files.py:20]: Would it be more clear if I change comment to this?
```py
# if you put widget into setup_app(), lona will not discover static files
```

Comment (fscherf) [reply tests/test_static_files.py:20]: Oh! That sounds like a bug. We should add a "FIXME: " then

Comment (maratori) [reply tests/test_static_files.py]: I've fixed comment.

I'm not sure it's a bug. However need to investigate for sure.
I'll create an issue for that.

File: lona/static_file_loader.py (hunk 1 above)
Comment (maratori) [reply lona/static_file_loader.py]: Fixed

File: tests/test_static_files.py (hunk 2 above)
Comment (fscherf) [reply tests/test_static_files.py]: Let's add the fixme anyway. The widget should be part of the setup stage, not the test i think

Comment (maratori) [reply tests/test_static_files.py]: Added

**Comment by fscherf** [comment]
@maratori Thanks! :)

**Pull request merged**